app/src/main/res/layout/list_playlist_item.xml

```diff
@@ -42,7 +42,7 @@
 
     <org.schabi.newpipe.views.NewPipeTextView
         android:id="@+id/itemTitleView"
-        android:layout_width="match_parent"
+        android:layout_width="230dp"
```

Comment:
What? You should not use fixed amounts... It should be as big as possible

(same in other places)